### PR TITLE
Include number by default, implement suffix method

### PR DESCRIPTION
**Pull request by ameykusurkar**
Addresses issue #2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-  i = Math.abs(i);
-  if (Number.isNaN(i)) throw new Error('i is not a number.');
+const thize = (input) => {
+  return input + suffix(input);
+};
+
+const suffix = (input) => {
+  const i = Math.abs(input);
+  if (Number.isNaN(i) || !Number.isInteger(i)) {
+    throw new Error("'" + input + '\' is not an Integer.');
```
Comment (ycmjason) [inline index.js]: Use template literal instead:

```javascript
throw new Error(`'${input}' is not an Integer.`);
```

File: index.js [hunk 2]
```diff
@@ -18,4 +24,5 @@ const thize = (i) => {
 
 const teen = i => 10 < i && i < 20;
 
-module.exports = thize;
+module.exports = exports = thize;
```
Comment (ycmjason) [inline index.js]: Don't think you have to do ` = exports`

Just leave it as

```js
module.exports = thize;
```

File: test/index.test.js [hunk 3]
```diff
@@ -82,22 +82,25 @@ const specs = [
...
-    input: -51,
-    expected: 'st',
-  },
 ];
 
 describe('thize', () => {
   specs.forEach(({ description, input, expected }) => {
-    it(description, () => assert.equal(thize(input), expected));
-    it(`Negative ${description}`, () => assert.equal(thize(-input), expected));
+    it(description, () => assert.equal(thize(input), input + expected));
```
Comment (ycmjason) [inline test/index.test.js]: Now the `expected` is not really the "expected" result. Would it make more sense if we rename `expected` to `suffix`/`expectedSuffix`?

**Review by ycmjason — CHANGES_REQUESTED** [review]
LGTM, left a few comments. 👍 🥂

File: test/index.test.js (hunk 3 above)
Comment (ameykusurkar) [reply test/index.test.js]: you mean in the `specs` array?

Comment (ycmjason) [reply test/index.test.js]: Ya, what do you think?

**Comment by ycmjason** [comment]
LGTM, Thank you!

**Pull request merged**